src/Shared/Navbar/NavBar.tsx

```diff
@@ -1,13 +1,18 @@
 import React, { useContext } from 'react'
 import logo from "../../assets/images/navLogo.png"
+import logo1 from "../../assets/images/pms.png"
 import avatar from "../../assets/images/avatar.png"
 import { AuthContext } from '../../Context/AuthContext'
+import { ThemeContext, ITheme } from '../../Context/ThemeContext';
+
 export default function NavBar() {
-  const { userData }: any = useContext(AuthContext)
+  const { userData }: any = useContext(AuthContext);
+  const { theme }:ITheme = useContext(ThemeContext);
   return (
-    <nav className="navbar navbar-expand-lg nav-bg bg-light p-0 ">
-      <div className="container-fluid ">
-        <img src={logo} alt="" />
+    <nav className="navbar navbar-expand-lg nav-bg p-0 ">
+      <div className="container-fluid p-0">
+        {theme ==='light' ?(<img src={logo} alt="" className='logo' />):(<img src={logo1} alt=""  className='logo' />)}
```

Comment:
if we changed the state as mentioned before to a boolean value, this could be also more simpler and easier to read as follows:
```
{isDarkMode && <img src={logo1} alt=""  className='logo' />}
{!isDarkMode && <img src={logo} alt="" className='logo' />}
```